samples/routing/custom/sample.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Custom routing
+summary: Customizing NServiceBus message routing
+component: Core
+tags:
+- Routing
+redirects:
+---
+
+
```

Comment:
Would be useful to provide short intro (1-2 paragrahps, few sentences) to explain what's the purpose of this sample and why would somebody use custom routing, what problem does it solve?